cmd/main.go

```diff
@@ -244,6 +244,12 @@ func electMaster(mgr manager.Manager, nsxClient *nsx.Client) {
 	log.Info("I'm trying to be elected as master")
 	<-mgr.Elected()
 	log.Info("I'm the master now")
+	// During the leader transitions, two operator may run in Kubernetes at the same time
+	// when the new one acquire the lease while the old one does not realize it loses the lease.
+	// To avoid the race condition during such period, make the new master operator sleeping
+	// for 15 seconds, which is the default value of the Leader Election Lease Duration.
+	log.Info("Sleeping for 15 seconds...")
```

Comment:
Waiting a 15-second delay to let the old instance know that it has lost its lease.

Reply:
Updated, thanks!